tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle@2019_10_29/stacktrace_enforce_min_frames.pysnap

```diff
@@ -8,23 +8,23 @@ app:
   component:
     app (exception of system takes precedence)
       exception (ignored because this variant does not have a contributing stacktrace, but the system variant does)
-        stacktrace (discarded because stacktrace only contains 1 frame which is under the configured threshold by grouping enhancement rule (family:native min-frames=2))
+        stacktrace (discarded because stacktrace only contains 1 frame which is under the configured threshold by stack trace rule (family:native min-frames=2))
```

Comment:
```suggestion
        stack trace (discarded because stack trace only contains 1 frame which is under the configured threshold by stack trace rule (family:native min-frames=2))
```